packages/code-du-travail-api/src/server/data_sources/cdtn_annuaire.js

```diff
@@ -0,0 +1,38 @@
+const elasticsearchClient = require("../conf/elasticsearch.js");
+
+const elasticsearchIndexName = "cdtn_annuaire";
+const elasticsearchTypeName = "cdtn_annuaire";
+
+/**
+ * Return a list of item matching the departement's code
+ *
+ * @returns {Object} An elasticsearch response.
+ */
+async function getItemsBydistance(params) {
+  const { distance = "30km", coord } = params;
```

Comment:
`30km` is a Magic value that should come from a Magic contant file :)
[Can be done in another PR]